tests/testthat/test-regressions.R

```diff
@@ -16,7 +16,7 @@ test_that("manually created label l rows are always visible",
 ## was error before rtables 0.3.2.16
 test_that("printing table with 0 rows works", {
     norows <- rtable(c("hi", "lo"))
-    expect_identical(print(norows), norows)
+    expect_identical(capture.output(print(norows)), capture.output(norows))
```

Comment:
this isn't teh same test, please do the same as elsewhere with `capture.output({prout <- print(tbl)})` ...